### ISSUE TITLE
Comment in property value cause error

### ISSUE THREAD

**Issue by thiakil** [sass/libsass#1422]
libsass version 3.2.5-11-g3565 (master)

`.next { right: /*-120*/-60px; }`

```
./sassc commenttest.scss
Error: invalid property name
        on line 1 of commenttest.scss
>> .next { right: /*-120*/-60px; }
   -------------^
```

Worked on 3.2.0-beta.6 (with node-sass)
```
Working version:
node-sass       3.0.0-beta.7
libsass         3.2.0-beta.6
```

**Comment by xzyfer** [comment]
Spec added https://github.com/sass/sass-spec/pull/465

**Comment by xzyfer** [comment]
This has been fixed and will be 3.3
